strapi/strapi
# Fix data transfer relations

### What does it do?

Accepts components also when transferring the links. This fixes the errors related to missing relations inside components after a data transfer (export and import or transfer)

### Why is it needed?

To allow relations inside of components to be transferred

### How to test it?

Create a content type with a component that has a relation to another content type inside it and add some data to that content type. Try to transfer or export and import to another Strapi project that has the same content type and everything should be transferred, as shown in [this Loom video](https://www.loom.com/share/27a1d2627261478088f91764cc34f771?sid=33193cbe-78d0-4e1f-9180-f1c330c7b5e5)

### Related issue(s)/PR(s)

Fixes https://github.com/strapi/strapi/issues/16813
Fixes https://github.com/strapi/strapi/issues/16811
Fixes https://github.com/strapi/strapi/issues/16891
Fixes https://github.com/strapi/strapi/issues/16999
Fixes https://github.com/strapi/strapi/issues/17343
